.github/workflows/release.yml

```
@@ -0,0 +1,25 @@
+name: Gem Release
+
+on:
+  create:
+    ref_type: tag
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'voxpupuli/beaker-puppet'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Ruby 2.6
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
```

Ruby 2.7 contains Rubygems 3.1.4
```suggestion
      - name: Install Ruby 2.7
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: '2.7'
```